spec/functional/command/repo/update_spec.rb

```diff
@@ -47,5 +47,21 @@ module Pod
       run_command('repo', 'update', 'repo2')
       (repo2 + 'README').read.should.include 'Updated'
     end
+
+    it 'repo updates do not fail when executed in parallel' do
+      repo1 = repo_make('repo1')
+      repo_clone('repo1', 'repo2')
+      repo_make_readme_change(repo1, 'Updated')
+      Dir.chdir(repo1) { `git commit -a -m "Update"` }
```

Comment:
`Pod::Executable.capture_command!('git', %w(commit -a -m Update))` ?

Reply:
I put this inside of the `chdir`, is that what you wanted? Why is this needed?

Reply:
It's better to avoid using the shell whenever possible